src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js

```diff
@@ -300,6 +194,14 @@ define(function() {
                     el: this.$find(constants.toolbarSelector),
                     instanceName: this.options.instanceName,
                     template: this.sandbox.sulu.buttons.get({
+                        add: {
+                            options: {
+                                class: null,
```

Comment:
the class-property of the add-button is set to highlighted by default, we dont want a highlighted add-button here